<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>620</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>620</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>291</width>
     <height>540</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>536</width>
     <height>540</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="background">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../assets/photos/pngwing.com.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>207</x>
     <y>0</y>
     <width>413</width>
     <height>540</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>1476457</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QStackedWidget::tab { display: none; }</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="optionPageMain">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>30</number>
     </property>
     <property name="leftMargin">
      <number>100</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>100</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="logo_imobiliare">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>65</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../assets/photos/Imobiliare.PNG</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loginMain">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: 2px solid transparent;
    border-radius: 15px;      
    padding: 8px;                  
    background-color: white;        
    color: black;             
    transition: background-color 0.3s ease; 
}

QPushButton:hover {
    background-color:  rgb(245, 245, 245);
    color: black;                  
}

QPushButton:pressed {
    background-color:  #d0d0d0;
    color: black;
}</string>
       </property>
       <property name="text">
        <string>Log In</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="registerMain">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: 2px solid transparent;
    border-radius: 15px;      
    padding: 8px;                  
    background-color: white;        
    color: black;             
    transition: background-color 0.3s ease; 
}

QPushButton:hover {
    background-color:  rgb(245, 245, 245);
    color: black;                  
}

QPushButton:pressed {
    background-color: #d0d0d0;
    color: black;
}</string>
       </property>
       <property name="text">
        <string>Register</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="logInPage">
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2,5,5,5,0,1,1">
     <property name="spacing">
      <number>28</number>
     </property>
     <property name="leftMargin">
      <number>50</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>50</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="emailLogIn">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	border: 2px solid transparent; 
    border-radius: 15px;
	padding: 5px;
}</string>
       </property>
       <property name="placeholderText">
        <string>Email</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="passwordLogIn">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	border: 2px solid transparent; 
    border-radius: 15px;
	padding: 5px;
}</string>
       </property>
       <property name="placeholderText">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="loginSubmit">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: 2px solid transparent;
    border-radius: 15px;      
    padding: 8px;                  
    background-color: white;        
    color: black;             
    transition: background-color 0.3s ease; 
}

QPushButton:hover {
    background-color:  rgb(245, 245, 245);
    color: black;                  
}

QPushButton:pressed {
    background-color:  #d0d0d0;
    color: black;
}</string>
       </property>
       <property name="text">
        <string>Log In</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="cancel_login">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: 2px solid transparent;
    border-radius: 15px;      
    padding: 8px;                  
    background-color: white;        
    color: black;             
    transition: background-color 0.3s ease; 
}

QPushButton:hover {
    background-color:  rgb(245, 245, 245);
    color: black;                  
}

QPushButton:pressed {
    background-color:  #d0d0d0;
    color: black;
}</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_9">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="optionPage">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="leftMargin">
      <number>100</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>100</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="clientButton">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: none;
    background-color: transparent;
}

QPushButton:hover {
    icon: url(:/icons/user_register_hovered.png);
}

QPushButton:pressed {
    icon: url(:/icons/user_register_hovered.png);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../icons/icons.qrc">
         <normaloff>:/icons/user_register.png</normaloff>:/icons/user_register.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>90</width>
         <height>90</height>
        </size>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="client_label">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Client</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_7">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="agentButton">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: none;
    background-color: transparent;
}

QPushButton:hover {
    icon: url(:/icons/real-estate-agent_hovered.png);
}

QPushButton:pressed {
    icon: url(:/icons/real-estate-agent_hovered.png);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../icons/icons.qrc">
         <normaloff>:/icons/real-estate-agent_register.png</normaloff>:/icons/real-estate-agent_register.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="agent_label">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Agent</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_6">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="back_option">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: 2px solid transparent;
    border-radius: 15px;      
    padding: 8px;                  
    background-color: white;        
    color: black;             
    transition: background-color 0.3s ease; 
}

QPushButton:hover {
    background-color:  rgb(245, 245, 245);
    color: black;                  
}

QPushButton:pressed {
    background-color:  #d0d0d0;
    color: black;
}</string>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_8">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="registrationPage">
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>480</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="leftMargin">
      <number>50</number>
     </property>
     <property name="topMargin">
      <number>20</number>
     </property>
     <property name="rightMargin">
      <number>50</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="titleRegister">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>24325</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string>Client Register</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="firstNameInput">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	border: 2px solid transparent; 
    border-radius: 15px;
	padding: 5px;
}</string>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>First Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lastNameInput">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	border: 2px solid transparent; 
    border-radius: 15px;
	padding: 5px;
}</string>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>Last Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="emailInput">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	border: 2px solid transparent; 
    border-radius: 15px;
	padding: 5px;
}</string>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>Email</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="passwordInput">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	border: 2px solid transparent; 
    border-radius: 15px;
	padding: 5px;
}</string>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="password_req1">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>10</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>• At least 8 characters</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="password_req2">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>15</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Century Gothic</family>
        </font>
       </property>
       <property name="text">
        <string>• At least one numeric digit (e.g., 0-9)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="password_req3">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>15</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Century Gothic</family>
        </font>
       </property>
       <property name="text">
        <string>• At least one special character (e.g., !, @, #, $,  &amp;, *)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="phoneInput">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	border: 2px solid transparent; 
    border-radius: 15px;
	padding: 5px;
}</string>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>Phone Number</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="submitButton">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: 2px solid transparent;
    border-radius: 15px;      
    padding: 8px;                  
    background-color: white;        
    color: black;             
    transition: background-color 0.3s ease; 
}

QPushButton:hover {
    background-color:  rgb(245, 245, 245);
    color: black;                  
}

QPushButton:pressed {
    background-color: #d0d0d0;
    color: black;
}</string>
       </property>
       <property name="text">
        <string>Register</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="cancel_register">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: 2px solid transparent;
    border-radius: 15px;      
    padding: 8px;                  
    background-color: white;        
    color: black;             
    transition: background-color 0.3s ease; 
}

QPushButton:hover {
    background-color:  rgb(245, 245, 245);
    color: black;                  
}

QPushButton:pressed {
    background-color:  #d0d0d0;
    color: black;
}</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <zorder>stackedWidget</zorder>
  <zorder>frame</zorder>
 </widget>
 <resources>
  <include location="../icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
